virttest/qemu_storage.py

```diff
@@ -345,10 +346,10 @@ def snapshot_del(self, blkdebug_cfg=""):
                                        " parameters")
         if blkdebug_cfg:
             cmd += " blkdebug:%s:%s" % (blkdebug_cfg, self.image_filename)
+            return process.run(cmd, ignore_status=True)
```

Comment:
I'm confused...
You want to supress Exception here.
But you catch it in test case.

If I'm wrong, Please point it out to me. Thanks in advance.